client/app/bundles/HelloWorld/containers/HelloWorld.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import HelloWorldWidget from '../components/HelloWorldWidget';
+import * as actions from '../actions/helloWorldActionCreators';
+
+// Which part of the Redux global state does our component want to receive as props?
+const mapStateToProps = (state) => ({ name: state.name });
+
+// Simple example of a React "smart" component
+const HelloWorld = (props) => {
```

Comment:
I agree, the extra SFC seems unnecessary. This is also how it's done in the official tutorial of Redux